### DISCUSSION TITLE
Can it run on NVIDIA Jetson Nano Orin hardware?

### DISCUSSION THREAD

**Discussion by mxzou** [Genesis-Embodied-AI/genesis-world#18] [General]
Hi! Just curious can it maybe run on Nvidia's newly released Jetson Orin Nano dev kit? RTX 4090 was mentioned but unsure if that's the definitive hard spec. Thank you very much for letting me know! This is their official [datasheet.pdf](https://github.com/user-attachments/files/18192006/datasheet.pdf) just in case.

Best
--MXZ

**Comment by mxzou** [comment]
P.S. Just to clarify, it's the 2025 batch Jetson Orin Nano Super that I'm interested in.

**Comment by lucpaquetlevio** [comment]
Same question. Can't wait to see your stuff running on that monster. https://blogs.nvidia.com/blog/jetson-generative-ai-supercomputer/

**Reply by rhystz** [reply]
This thing isnt so much a compute monster as it is a compact and cost effective way to provide edge compute (and llms etc) to devices that interact with the real world. ie things that are or might get far away from your workstation.

The synergy of Genesis and the jetsons would be in training and visualising training for robotics with genesis, and then implementing the training to your robot with an onboard jetson.
